### PR TITLE
Support for GCC 6

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by hernando** [comment]
This is a bit weird, I've been able to compile many of our tools with gcc 6.0 without making any changes. Which is the system in which you're having problems.

**Comment by eile** [comment]
Hold this one. Won't work on Ubuntu

**Comment by eile** [comment]
More better. Proceed at will.

File: CommonCompiler.cmake [hunk 1]
```diff
@@ -41,6 +41,11 @@ endif()
 option(COMMON_WARN_DEPRECATED "Enable compiler deprecation warnings" ON)
 option(COMMON_ENABLE_CXX11_STDLIB "Enable C++11 stdlib" OFF)
 option(COMMON_DISABLE_WERROR "Disable -Werror flag" OFF)
+if($ENV{CMAKE_COMMON_USE_CXX03_ABI})
```
Comment (eile) [inline CommonCompiler.cmake]: erm yes. Set by env module.

**Pull request merged**